docs/commandline/create.md

```diff
@@ -0,0 +1,57 @@
+# create
+
+create command is used to create a container according to user's configuration.
+
+## Description
+
+The `pouch create` command creates a container object in Pouchd. When creating, all configuration user input will be stored in memory store of Pouchd. This is useful when you hope to create a container configuration ahead of time so that Pouchd will preserve the resource in advance. The container you created is ready to start when you need it.
+
+## Help Information
+
+``` shell
+$ pouch create --help
+Create a new container with specified image
+
+Usage:
+  pouch create [image] [flags]
+
+Flags:
+  -h, --help             help for create
+      --name string      specified the container's name
+  -t, --tty              allocate a tty device
+  -v, --volume strings   create container with volumes
+
+Global Flags:
+  -H, --host string        Specify listen address of pouchd (default "unix:///var/run/pouchd.sock")
+      --tlscacert string   Specify CA file of tls
+      --tlscert string     Specify cert file of tls
+      --tlskey string      Specify key file of tls
+      --tlsverify          Switch if verify the remote when using tls
+```
+
```

Comment:
As the pouch CMD is under developing, could we just write e.g. "Use pouch help to get usage"  instead of paste the output of usage. In case the usage is changed, we will need little work to update the corresponding doc.

Reply:
Yes, that is one kind of ways. What you said will definitely reduce cost of that we maintain the pouch's documentation. While to be honest, I don't think this is what our users wish to see. They wish to see the complete and detailed documents almost without any practice. Since 
 when pouch is open sourced, we received lots of feedback that users are looking forward to seeing the sufficient document for guidance. So I stand for adding this part.

Considering that maintaining docs will cost us lots of time, I have a following suggestion:
1. We add command help as usual;
2. We regularly update these message in doc every two weeks or less;
3. Maybe we could advocate more brilliant guys from community to be engaged in.

WDYT? @Letty5411

Reply:
OK, this makes sense.